daemon/daemon_solaris.go

```diff
@@ -84,6 +320,16 @@ func (daemon *Daemon) platformReload(config *Config) map[string]string {
 
 // verifyDaemonSettings performs validation of daemon config struct
 func verifyDaemonSettings(config *Config) error {
+
+	if config.DefaultRuntime == "" {
+		config.DefaultRuntime = stockRuntimeName
```

Comment:
`stockRuntimeName` would need to be moved into the platform specific `daemon/config_*.go` unless you're planning to  call it `runc` under `Solaris` :wink: